src/app/client/src/app/modules/groups/components/activity/activity-search/activity-search.component.ts

```diff
@@ -362,6 +363,14 @@ export class ActivitySearchComponent implements OnInit, OnDestroy {
     };
   }
 
+  private listenLanguageChange() {
```

Comment:
@sedin-tushar please add the comment description of why we are using this method 

sample reference:

  /**
   * @description  write your comment description here.
   */